packages/core/src/useFormGroup/useFormGroup.ts

```diff
@@ -66,7 +66,17 @@ export function useFormGroup<TInput extends FormObject = FormObject, TOutput ext
 ) {
   const id = useUniqId(FieldTypePrefixes.FormGroup);
   const props = normalizeProps(_props, ['schema']);
-  const getPath = () => toValue(props.name);
+  const pathPrefixer = usePathPrefixer();
+
+  const getPath = () => {
+    const path = toValue(_props.name);
+
+    const prefixPath = pathPrefixer ? pathPrefixer.prefixPath(path) : path;
+
+    if (!prefixPath) return path;
```

Comment:
Let's add a block here, I should probably add an eslint rule for it.
```suggestion
    if (!prefixPath) {
      return path;
    }
```